src/NEO/CoinReference.h

```diff
@@ -7,7 +7,7 @@
 #pragma once
 
 #include "../uint256.h"
-#include "Data.h"
```

Comment:
hmm, This should just work

Reply:
Yes, just want to keep same style with others.

Reply:
Funny, if you look at the last change to this file, "../Data.h" was changed to "Data.h", but across the whole project.
https://github.com/trustwallet/wallet-core/pull/2563
I suggest to change to without the ".." format the other includes as well.

Reply:
Awesome, I will fix it.